### PR TITLE
[Extensions.Docker] Nullable

### PR DESCRIPTION
**Pull request by Kielek**
Towards #894

## Changes

Nullable for docker and tests.

For significant contributions please make sure you have completed the following items:

* ~~[ ] Appropriate `CHANGELOG.md` updated for non-trivial changes~~
* ~~[ ] Design discussion issue #~~
* ~~[ ] Changes in public API reviewed~~

### CODE REVIEW

**Comment by Kielek** [comment]
@iskiselev, it will be great if you can check.

**Review by cijothomas — APPROVED** [review]

File: src/OpenTelemetry.Extensions.Docker/.publicApi/net462/PublicAPI.Shipped.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+#nullable enable
```
Comment (iskiselev) [inline src/OpenTelemetry.Extensions.Docker/.publicApi/net462/PublicAPI.Shipped.txt:1]: I don't know much about PublicAPI.Shipped.txt and PublicAPI.Unshipped.txt, but it looks strange that there is
```
nullable enabled
```
here, but not in Unshipped. It feels that it should be either in both or in none.

**Comment by iskiselev** [comment]
Added one question, but LGTM otherwise.

File: src/OpenTelemetry.Extensions.Docker/.publicApi/net462/PublicAPI.Shipped.txt (hunk 1 above)
Comment (Kielek) [reply src/OpenTelemetry.Extensions.Docker/.publicApi/net462/PublicAPI.Shipped.txt:1]: It is the way how VS is generating it.
It is enough to make it in one place and usually we put it to Shipped.

**Pull request merged**